src/Wrapper.js

```diff
@@ -311,6 +333,102 @@ class Wrapper extends React.PureComponent {
   }
 }
 
+const SwipeContainer = (() => {
+  let _progress = 0
+
+  const SwipeContainer = ({
+    children,
+    enabled,
+    menuPanelOpened,
+    onMenuPanelClose,
+    onMenuPanelOpen,
+    width,
+  }) => {
+    const oneThirdWindowWidth = width / 3
+
+    if (!enabled) {
+      return <Container>{children(Number(menuPanelOpened))}</Container>
+    }
+
+    return (
+      <Gesture passive={{ passive: true }}>
+        {({
+          delta: [xDelta, yDelta],
+          direction: [xDir, yDir],
+          down,
+          event,
+          initial: [xInitial],
+          xy: [x],
+        }) => {
+          if (!down && _progress > 0 && _progress < 1) {
```

Comment:
That could only happen if the user dragged beyond the screen (which means they've released)

Reply:
Once we do the fix to have the touch point exactly on the border, `1` should happen before the touch point gets outside of the screen:

<img width=240 src="https://scri.ch/181x.png">

Reply:
The `delta` is being calculated considering the touch move distance vs screen width.

Reply:
Yes, which is a problem because the panel doesn’t follow the touch point (see [this comment](https://github.com/aragon/aragon/pull/606#discussion_r257082487)), but once the fix would be applied, `1` would be positioned as show in the drawing ⬆️, and it should be possible to get beyond it and slide towards the left again after that.